### ISSUE TITLE
Подтверждение использования шифрования OMEMO в чате

### ISSUE THREAD

**Issue by OmlineEditor** [miranda-ng/miranda-ng#3096]
Прошу добавить в интерфейс возможность видеть что сообщение было переслано с использованием шифрования OMEMO.
В миранде для OTR шифрования уже есть символ в начале каждого сообщения и зеленая иконка на панеле которая подветрждает что сообщение прислали с шифрованием, прошу сделать аналогичное и для OMEMO.

**Comment by Neustradamus** [comment]
@OmlineEditor: Your request has been done?

**Comment by OmlineEditor** [comment]
yes, now I see a lock near the message if encryption is taking place. the icon is used from the library MirOTR.dll
A blue icon is used to indicate that the message is encrypted. if the message is not encrypted, nothing is shown.

please replace the icon for the encrypted message with a green lock, so it is more understandable. I also ask you to add an analog of a gray lock with an open shackle for unencrypted messages, add a new red lock for an unencrypted message.

![blue](https://github.com/miranda-ng/miranda-ng/assets/13348933/89c5d2f9-5981-4ba0-8471-0c944759bab3)
